pkg/login/social/github_oauth.go

```diff
@@ -48,7 +48,8 @@ func (s *SocialGithub) IsSignupAllowed() bool {
 }
 
 func (s *SocialGithub) OrgToRoleMap() map[string][]SocialGroup {
-  return nil
+	var emptyMap map[string][]SocialGroup
```

Comment:
Add a comment to these to make it clear that group mapping is unimplemented for this provider.

Reply:
Added.